.gitignore

```diff
@@ -7,3 +7,8 @@ target/
 *.iml
 /.idea/
 /.vscode/
+
+# artifacts from npm
+node_modules/
+package.json
+package-lock.json
```

Comment:
What kind of operation will create these files?

I believe this also ignores `website/package.json` and `website/package-lock.json`, which is not what we want.

Reply:
Installing & using prettier is what got me those 2 files and a fat `node_modules` folder. Excluding them shouldn't be a problem, I checked and the files in `website/` aren't excluded.

Reply:
Why would you want prettier?

If you are using it to format website, it should be installed in the `website` directory, which it already comes with prettier.

> I checked and the files in website/ aren't excluded.

This is because it is already marked as tracking. Try `git rm website/package.json` and it should no longer be tracked after that.